### ISSUE TITLE
Accessibility audit

### ISSUE THREAD

**Issue by mitchdowney** [podverse/podverse-rn#453]
The app is almost definitely not accessible right now :( any help auditing the app and making accessibility recommendations would be greatly appreciated.

**Comment by mitchdowney** [comment]
Going to replace this audit issue with an implementation issue.
